.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+## Issue #<number>
```

Comment:
issue number is assigned itself, no need to mention it here

Reply:
Copy paste really got me here! 🤣

Reply:
Done.